manifests/09_deployment.yaml

```diff
@@ -85,3 +85,16 @@ spec:
             requests:
               cpu: 10m
               memory: 20Mi
+        - name: vsphere-problem-detector
+          image: quay.io/openshift/origin-vsphere-problem-detector:latest
+          command:
+          - start
+          - --listen=0.0.0.0:6456
```

Comment:
Either use `args: start ...` or `command: vsphere-problem-detector start ...`
(I got stuck at this for non-trivial time :-)